.github/workflows/_test_upstream_t5x.yaml

```diff
@@ -174,8 +192,47 @@ jobs:
   t5x-multi-node:
     strategy:
       matrix:
-        N_GPU: [1, 2, 4, 8]
-        N_NODE: [1, 2]
+        include:
+          - TEST_NAME: "1N1G"
```

Comment:
There are some assumptions that the test names should match [their older names ](https://github.com/NVIDIA/JAX-Toolbox/tree/main/.github/workflows/baselines/T5X_MGMN/upstream)so that baselines are found. Do you mind permuting the nodes and gpus? So `2N8G` -> `8G2N`?